README.md

```diff
@@ -33,7 +33,21 @@ To enable testing in the app, your Rails app needs:
 the dimension to use in Google Analytics
 3. A response HTTP header that tells Fastly you're doing an A/B test
 
-Let's say you have this controller:
+Start by defining which acceptance testing framework you will use. This gem
+supports both Capybara and ActiveSupport. In order to configure it, add this to
+your test helper file:
+
+```
+GovukAbTesting.configure do |config|
+  config.acceptance_test_framework = :capybara # or :active_support
```

Comment:
Is there a default? If not, I think there should be. If so, please document which one it is.